package.json

```diff
@@ -35,20 +35,20 @@
     "release": "npm run release:coverage && npm run release:plugin"
   },
   "peerDependencies": {
-    "prettier": "^2.0.0"
+    "prettier": "^2.1.2"
```

Comment:
Please keep this one at `^2.0.0` so that it's compatible with all versions between >= 2.0.0 & < 3.0.0.

But you can update the prettier dependency below to `2.1.2`.

Reply:
Will do.